src/pysight/nd_hist_generator/movie.py

```diff
@@ -284,6 +284,7 @@ def __create_memory_output(
         self.stack[channel].append(data)
         assert len(data.shape) > 2
         self.summed_mem[channel] += np.uint16(data.sum(axis=0))
+        self.flim_df[channel] = flim_hist
```

Comment:
The docstring here is wrong, but also the logic - how do you know there would be a FLIM histogram? And what's `self.flim_df`? I don't think it belongs here.